dash/development/_py_components_generation.py

```diff
@@ -551,7 +576,7 @@ def map_js_to_py_types_flow_types(type_object):
             else ""
         ),
         # React's PropTypes.shape
-        signature=lambda indent_num: "dict containing keys {}.\n{}".format(
+        signature=lambda indent_num: "dict with keys: {}.\n{}".format(
```

Comment:
One last thing: Flow isn't used much but since we have it lets simplify it the same as react proptypes: remove the list of keys here since they're enumerated below, and remove the "Those keys have the following types" line.